### PR TITLE
Feature/update for page changes

### PR DESCRIPTION
**Pull request by petergoldstein**
Change XPath used to grab IP and host list to account for HTML changes.

### CODE REVIEW

File: spec/lib/hidemyass/ip_spec.rb [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 describe HideMyAss::IP do
 
-  # Tipically we would use webmock here to fake the http request,
+  # Typically we would use webmock here to fake the http request,
```
Comment (jassa) [inline spec/lib/hidemyass/ip_spec.rb:5]: oops didn't catch that typo

**Comment by jassa** [comment]
Thanks!

**Comment by petergoldstein** [comment]
@jassa BTW, filtering is no longer working after the latest site changes.  I'm looking at it now, but any help would be appreciated.

**Comment by jassa** [comment]
thanks for the heads up, I'll take a look at it tonight

**Pull request merged**